content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Digital Twin Manager
+
+Cumulocity IoT Digital Twin Manager is a layered product hosted on Cumulocity platform. The Digital Twin Manager enables you to create and manage assets around your physical connected devices in Cumulocity IoT. Assets can be used to structure and describe devices in logical hierarchies as encountered in the real world environment.
+
+The Digital Twin Manager consists of several components, which support you to create your assets and asset hierarchies:
+
+    •	Property library - Create your own custom properties (simple or complex) to define how data should be stored on an asset
+    •	Asset types - Define asset types to describe the blueprint of your assets
+    •	Localization - Dynamically translate asset type and custom property descriptions to multiple languages
+    •	Assets navigator - Manage your asset hierarchies
+
+You can access Digital Twin Manager from the Cumulocity IoT app switcher:
+
+![App switcher](/images/dtm/app-switcher-dtm.png)
+
+See [DTM User Guide](<<TBD>>) for the complete documentation.
+
+**Property Library**
+
+The Property Library can be used to create your own custom properties to define how data should be stored on an asset. These custom properties can be either complex (nested object) or simple (key-value pair). Furthermore, you can choose a specific data type for each property you define. As of now, the Property Library supports following data types: Text, Number, Date Picker, Enumeration, Boolean.
+
+**Asset Types**
+
+Asset Types serve as a blueprint for your assets and define how they are structured. An Asset Type defines what custom properties an asset should have. Additionally, you can specify relationships between multiple Asset Types, which will be respected when you create your actual asset hierarchies. Once the Asset Type is created, it can be used to create various assets.
+
+**Assets Navigator**
+
+The Assets Navigator is the starting point to create your asset hierarchies based on the asset types and custom properties, you have defined previously. Once you have created your asset hierarchies you can use the Assets Navigator to view and manage them.
+
+**Localization**
+
+You can use the Localization component to define translations for your asset types and custom properties. These translations can be provided for all supported languages in Cumulocity IoT. This component requires to have the feature-branding feature subscribed for your Cumulocity tenant.
+
+### Installation
```

Comment:
Needs one more # to have the correct heading level

Reply:
Added one more #